numpy/random/src/pcg64/pcg64.h

```diff
@@ -229,17 +229,6 @@ static inline void pcg_cm_step_r(pcg_state_setseq_128 *rng) {
 #endif
 }
 
-static inline uint64_t pcg_output_cm_128_64(pcg128_t state) {
```

Comment:
You might have seen better performance using `__forceinline` in place of `inline`.  AFAIK `__forceinline` overrides the compilers preferences, while `inline` doesn't.

Reply:
We have a `#define inline __forceinline` for `_WIN32` up at the top.